app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/AddButtonVisibility.java

```diff
@@ -0,0 +1,26 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+public class AddButtonVisibility extends EspressoTestBase {
+
+    @Test
+    public void testAAdd20Buttons() {
+        //Step 1. Add 20 items
```

Comment:
👎

Reply:
I changed the test name : "testAddButtonVisibility".